test/test-applications/integrations/TestApplication.GraphQL/Properties/launchSettings.json

```diff
@@ -12,20 +12,9 @@
       "commandName": "Project",
       "launchBrowser": false,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development",
```

Comment:
What is the reason to remove easy way to attach OTel Auto?

Reply:
part of cleanup, outdated values - actually most of our launchsettings are clean, so I just followed the pattern there.